integration-tests/jq/src/main/resources/application.properties

```diff
@@ -1,20 +0,0 @@
-## ---------------------------------------------------------------------------
-## Licensed to the Apache Software Foundation (ASF) under one or more
-## contributor license agreements.  See the NOTICE file distributed with
-## this work for additional information regarding copyright ownership.
-## The ASF licenses this file to You under the Apache License, Version 2.0
-## (the "License"); you may not use this file except in compliance with
-## the License.  You may obtain a copy of the License at
-##
-##      http://www.apache.org/licenses/LICENSE-2.0
-##
-## Unless required by applicable law or agreed to in writing, software
-## distributed under the License is distributed on an "AS IS" BASIS,
-## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-## See the License for the specific language governing permissions and
-## limitations under the License.
-## ---------------------------------------------------------------------------
-
-# https://github.com/apache/camel-quarkus/issues/4011
-#camel.context.global-options[CamelJacksonEnableTypeConverter] = true
-#camel.context.global-options[CamelJacksonTypeConverterToPojo] = true
```

Comment:
Please keep this file. I'm planning to restore these properties when the fix for #4011 arrives.

Reply:
I've restored the file.